packages/dialtone-vue2/components/split_button/split_button.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <span
+    data-qa="dt-split-button"
+    class="d-split-btn"
+    :style="{ width: width }"
+  >
+    <dt-button
+      v-dt-tooltip="alphaTooltipText"
+      data-qa="dt-split-button-alpha"
+      :size="size"
+      :class="[`d-split-btn__alpha d-split-btn__alpha--${size}`]"
+      :active="alphaActive"
+      :aria-label="alphaAriaLabel"
+      :icon-position="alphaIconPosition"
+      :label-class="alphaLabelClass"
+      :loading="alphaLoading"
+      :assertive-on-focus="assertiveOnFocus"
+      :disabled="disabled"
+      :importance="importance"
+      :kind="kind"
+      @click="(e) => handleClick('alpha', e)"
+    >
+      <template #icon>
+        <!-- @slot Alpha (left) button icon slot -->
+        <slot name="alphaIcon" />
+      </template>
+      <!-- @slot Default content slot -->
+      <slot name="default" />
+    </dt-button>
+    <!-- @slot Omega (right) content slot -->
```

Comment:
I guess here is where we actually explain which one is Alpha(left) and Omega(right).

O don't see necessary to have all these `<!--` comments.
The slot name should be descriptitable enough

Reply:
This comments are necessary to automatically generate the components documentation so we can use it on our documentation site.

Reply:
🥴🥴🥴🥴